lib/preload.js

```diff
@@ -88,8 +87,26 @@ const createContainer = (docker, options) => {
       `HTTPS_PROXY=${options.proxy || ''}`,
       `DOCKER_PORT=${options.dockerPort || ''}`
     ],
-    Mounts: mounts
-  })
+    HostConfig: {
+      Privileged: true,
+      Mounts: mounts
+    }
+  }
+  if (os.platform() === 'linux') {
+    containerOptions.HostConfig.NetworkMode = 'host'
+  } else {
+    if (options.dockerPort) {
+      containerOptions.ExposedPorts = {}
+      containerOptions.ExposedPorts[`${options.dockerPort}/tcp`] = {}
+      containerOptions.HostConfig.PortBindings = {}
+      containerOptions.HostConfig.PortBindings[`${options.dockerPort}/tcp`] = [{
+        HostPort: `${options.dockerPort}`,
+        HostIp: ''
+      }]
+    }
+    containerOptions.HostConfig.NetworkMode = 'bridge'
+  }
```

Comment:
I'm assuming this is tested on both mac+linux ?

Reply:
Actually I only tested on Linux.
This PR is supposed to make preload work on macs.
Could you please test it on a mac? I don't have one.

Reply:
@shaunmulligan are you able to help us here please?